lib/ui/window.dart

```diff
@@ -515,8 +515,15 @@ class Locale {
 
 /// The most basic interface to the host operating system's user interface.
 ///
+/// It exposes the size of the display, the core scheduler API, the input event
+/// callback, the graphics drawing API, and other such core services.
+///
 /// There is a single Window instance in the system, which you can
-/// obtain from the [window] property.
+/// obtain from `WidgetsBinding.instance.window`.
+///
+/// There is also a [window] singleton object in `dart:ui` if `WidgetsBinding`
+/// is unavailable. But we strongly advice to avoid statically referencing it.
```

Comment:
I think this update looks better. Thanks. One last detail, "advice" -> "advise".

Reply:
Thank you! Done.